typings/optimist/optimist.d.ts

```diff
@@ -33,7 +33,7 @@ declare module "optimist" {
 
         describe(key: string, desc: string): Optimist;
 
-        options(obj: {[key: string]: {alias: string, describe: string, default?: any}}): Optimist;
+        options(obj: {[key: string]: {alias?: string, describe: string, default?: any}}): Optimist;
```

Comment:
I manually edited this - arguably a bug in the typings. There's a pretty big TSD backlog (PRs take a long time to get accepted) so I didn't feel like making a PR, but probably should of.